src/main/java/com/corndel/cashregister/Level5.java

```diff
@@ -28,7 +31,34 @@ public class Level5 {
    */
   public static List<Item> transaction(int cost, List<Item> paid, List<Item> drawer) {
     // TODO
-    return null;
+
+    int totalPaid = 0;
```

Comment:
Good job getting this to work! When you get a chance, have a go and see if you can refactor it with the methods you have written in the other classes.